integrations/amazonKeys/cli/init.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash -e
+
+RESOURCE_NAME=$1
+SCOPE=$2
+
+if [ "$RESOURCE_NAME" == "" ]; then
+  echo "Usage: ./init.sh <resource_name> [scope]"
+  exit 1
+fi
+
+# If a scope is not passed, default to configure.
```

Comment:
Discussion on this behavior here: https://github.com/Shippable/heap/issues/1278#issuecomment-320001533. This is currently being handled in genExec, but will need to move to the scripts as genExec will not know the default scope of every integration.